notes.txt

```diff
@@ -20,3 +20,28 @@ UI Elements the we need to wireframe -
 --------> When videos are played, they pop up as a centered overlay? 
 
 
+
+This is our movie details component
+It is a Class-Component
+It should display all of the details about a movie, including:
+
+MovieDetails Component
+---> Backdrop (presented as a kind of Billboard-style header?)
+---> Tagline (if exists, overlaid on backdrop)
+
+Ratings
+---> Poster
+---> Title
+---> Release Date
+---> Avarage Rating
+---> User Rating
+
+Details
+---> Overview (Synopsis?)
+---> Genre (single or list)
+---> Budget
+---> Revenue
+---> Runtime
```

Comment:
So these would each be different components?

Reply:
The idea was that the MovieDetails component would have four sub-components: Billboard, Ratings, Details and Videos (which would contain all the Video components). @khalidwilliams would you rather see us render all of this as JSX in MovieDetails component? We'd love your feedback!!

##### Reasons we did this:
- Modularity of testing
- Organization / Readability / Maintenence
- We thought of this because of the Twitter example where page-view is made up of multiple components. Our movie details view will be a separate page in the application and it made sense for us to break it up like this.